Fix/bpo 21 responsive feature for community

VishalThapaliya opened the pull request:
Documentation on how to create a responsive application with Bonita Community edition.

Covers [BPO-21](https://bonitasoft.atlassian.net/browse/BPO-21)

VishalThapaliya committed:
documentation on responsiveness for community users

[Covers](https://bonitasoft.atlassian.net/browse/BPO-21)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonitasoft/bonita-doc?pullRequest=785) <br/>All committers have signed the CLA.

amottier reviewed (commented):
I would just includes a link to the Boostrap grid system documentation page (https://getbootstrap.com/docs/4.0/layout/grid/) for users who want to learn more about it.

abirembaut commented:
By the way (could be reused) : https://community.bonitasoft.com/blog/build-responsive-ui-designer-pages

VishalThapaliya committed:
documentation on responsiveness for community users

- a link to Bootstrap grid system documentation added

Covers [BPO-21](https://bonitasoft.atlassian.net/browse/BPO-21)

VishalThapaliya commented:
A link to bootstrap grid system documentation has been added.
